lib/zora/getZoraPfpLink.ts

```
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const getZoraPfpLink = (profile: any) => {
+    if (profile?.avatar && profile.avatar.startsWith("/")) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)